.github/workflows/main.yml

```diff
@@ -43,8 +43,11 @@ jobs:
 
       - name: Install/Link Postgres
         run: |
-          sudo apt update -yq
-          sudo apt install -yq --no-install-suggests --no-install-recommends postgresql-common postgresql-11 postgresql-client-11
+          sudo apt-get install curl ca-certificates gnupg
+          curl https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+          sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list'
+          sudo apt-get update
+          sudo apt-get install postgresql-11 postgresql-client-11
```

Comment:
You changed to `apt-get` and removed `-yq`, etc. Was there a reason?

Reply:
Not necessary when using `apt-get` I guess (note build clears postgres setup).